libs/common/src/assets/i18n/de.json

```diff
@@ -0,0 +1,892 @@
+	"account.code_saved": "Code eingegeben",
+	"account.config": "Kontoeinstellungen",
+	"account.consuming": "Konsumierend",
+	"account.create_new_db": "Erstellen Sie ein neues Nuclia DB-Konto",
+	"account.create_nua_key": "Erstellen Sie einen neuen Nuclia Understanding-API-Schlüssel",
+	"account.day_left": "Noch 1 Tag",
+	"account.days_left": "{{days}} Tage verbleiben",
+	"account.db_zone_description": "Wählen Sie die Zone und den Verarbeitungsanbieter aus, den Sie für dieses Konto verwenden möchten (standardmäßig die Zone des Kontos).",
+	"account.delete.error": "Fehler beim Löschen des Kontos",
+	"account.delete.go_to": "Wenn Sie Ihr Konto löschen möchten, gehen Sie bitte zu",
+	"account.delete.keep_user_title": "Möchten Sie Ihr Benutzerprofil behalten?",
+	"account.delete.subscription_detected": "Aktives Abonnement erkannt",
+	"account.delete_account": "Konto löschen",
+	"account.delete_account_confirm": "Ich möchte das Konto {{account}} und alle damit verbundenen Daten löschen.",
+	"account.delete_account_confirm_title": "Konto löschen?",
+	"account.delete_account_warning_1": "Durch das Löschen des Kontos {{account}} werden alle vorhandenen Daten gelöscht",
+	"account.delete_account_warning_2": "das Konto sowie etwaige Konfigurationsdaten, Benutzer- und Rollenzuweisungen",
+	"account.delete_account_warning_3": "jede Wissensbox und alle ihre Daten. Einschließlich der in NucliaDB indizierten Daten",
+	"account.delete_account_warning_4": "jedes trainierte Modell einer beliebigen Wissensbox",
+	"account.delete_user": "Benutzer löschen?",
+	"account.delete_user_confirm": "Möchten Sie Ihren Benutzer und den Zugang zu Nuclia löschen?",
+	"account.delete_user_warning": "Möchten Sie den Benutzer {{ username }}wirklich löschen?",
+	"account.edit_nua_key": "Bearbeiten Sie den API-Schlüssel für das Verständnis von Nuclia",
+	"account.email_info": "Mit dem Schlüssel verknüpfte Kontakt-E-Mail-Adresse",
+	"account.enhanced_model": "Erweitertes Modell",
+	"account.existing_keys": "Vorhandene Schlüssel",
+	"account.expired": "Abgelaufen",
+	"account.extend_trial": "Verlängern Sie die kostenlose Testversion",
+	"account.free_trial": "Kostenlose Testphase",
+	"account.free_trial_ended.description": "Die 15-tägige Testversion ist abgelaufen und Ihr Konto wurde gesperrt. Sie können entweder einen unserer Pläne abonnieren oder uns für eine Verlängerung der Testversion kontaktieren.",
+	"account.free_trial_ended.title": "Oh! Ihre kostenlose Testversion ist beendet",
+	"account.full_name": "Name und Nachname",
+	"account.generate_key": "Generieren Sie einen neuen Schlüssel",
+	"account.home": "Benutzerkonto Übersicht",
+	"account.invited_user": "Wir haben eine Einladungs-E-Mail an {{user}} gesendet, um diesem Konto beizutreten",
+	"account.kb-description": "Eine Knowledge Box ist eine Sammlung von Inhalten, nach denen Nuclia sucht. Einige Suchmaschinen nennen es „Index“ oder „Korpus“. In den meisten Fällen benötigen Sie für Ihre Bewerbung nur eine Knowledge Box.",
```

Comment:
Probably want to be consistent with either `Knowledge Box` or `Wissensbox`

Reply:
We want to be consistent for sure. Which one do you think would be the best option? I'll then make sure to use the same one everywhere.

Reply:
I´m not entirely sure. German speakers are pretty used to English words, but at the same time Wissensbox is pretty short and catchy. I think in the end it really depends if Knowledge Box is some kind of brand thing for us.

Reply:
I agree, and actually we should be consistent in all languages: I don't know if this was translated in spanish/catalan or not. I'll double check with Product team.